src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,40 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} key={movie.imbdId} />
+    ))}
+  </div>
 );
+
+MoviesList.propTypes = {
+  movies: PropTypes.arrayOf(
+    PropTypes.shape({
+      title: PropTypes.string.isRequired,
+      description: PropTypes.string.isRequired,
+      imgUrl: PropTypes.string.isRequired,
+      imdbUrl: PropTypes.string.isRequired,
+      imdbId: PropTypes.string.isRequired,
+    }),
+  ),
+};
+
+MoviesList.propTypes = {
+  movies: PropTypes.arrayOf(
+    PropTypes.shape({
+      title: PropTypes.string.isRequired,
```

Comment:
Basically, you've written here `MovieCard.propTypes` so next time you should write all `proptypes` in a single file, i. e., `types.js`. It would be easier for you to manage all of them.

Besides. Earlier, you've written:
```
MovieCard.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string,
  imgUrl: PropTypes.string.isRequired,
  imdbUrl: PropTypes.string.isRequired,
};
```
where `description` property is not required while here, it's required.